yarn.lock

```diff
@@ -1245,6 +1245,11 @@
   resolved "https://registry.yarnpkg.com/@types/minimatch/-/minimatch-3.0.3.tgz#3dca0e3f33b200fc7d1139c0cd96c1268cadfd9d"
   integrity sha512-tHq6qdbT9U1IRSGf14CL0pUlULksvY9OZ+5eEgl1N7t+OA3tGvNpxJCzuKQlsNgCVwbAs670L1vcVQi8j9HjnA==
 
+"@types/modernizr@^3.5.3":
```

Comment:
Because I only just added the typescript definitions dependency in this PR: https://github.com/vector-im/riot-web/pull/13032/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R99